test/config.test.js

```diff
@@ -356,6 +356,31 @@ describe('service worker config', () => {
     });
 });
 
+describe('copy config', () => {
+
+    it('copy config should be set to an empty object', () => {
```

Comment:
Nit: Remove "to an empty object" — just means that it's one less thing to remember to change if the default value changes.